_profiles/v2-article.html

```diff
@@ -38,6 +38,10 @@
 #
 budgets:
   -
+    metric: TTFB
```

Comment:
yep, I just thought it was a quick way of analysing (in slack) whether the alerts from the other metrics were likely due to this one, without having to go and check it out. Happy to remove it if you'd rather